flintrock/scripts/download-hadoop.py

```diff
@@ -18,16 +18,27 @@
 
 if __name__ == '__main__':
     hadoop_version = sys.argv[1]
-    apache_mirror_url = "http://www.apache.org/dyn/closer.lua/hadoop/common/hadoop-{v}/hadoop-{v}.tar.gz?as_json".format(v=hadoop_version)
+    custom_mirror_url = sys.argv[2]
```

Comment:
Hmm... It looks like `custom_mirror_url` will always be populated, even if the download setting is left to the default. No? I'm looking over at [this line](https://github.com/nchammas/flintrock/pull/118/files#diff-2866cac17a48e5c58161518d728d0f8fR133).